hooks/test

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -ex
+
+# Create a container with test and production postgres databases.
+docker pull postgres:9.5;
+docker run -d --name pg postgres:9.5;
+
+# Sleep to let PostgreSQL start up.
+sleep 10;
+
+# Create production and test databases.
+docker exec -u postgres pg psql -c "create user simplified with password 'test';";
+docker exec -u postgres pg psql -c "create database docker_prod;";
+docker exec -u postgres pg psql -c "create database docker_test;";
+docker exec -u postgres pg psql -c "grant all privileges on database docker_prod to simplified;";
+docker exec -u postgres pg psql -c "grant all privileges on database docker_test to simplified;";
+
+# Find the Postgres host IP address dynamically.
+pghost=$(docker inspect pg --format="{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}")
+echo "Postgres IP: $pghost"
```

Comment:
Previously I was making assumptions about the IP of the first-started container. It looks like there may be in an agent container in Docker Cloud or some-such, and this was no longer the case. Now we find the Postgres host IP dynamically.